src/Features/Core/Portable/NavigateTo/INavigateToSearchResult.cs

```diff
@@ -22,6 +22,7 @@ internal interface INavigateToSearchResult
     string? Summary { get; }
 
     INavigableItem NavigableItem { get; }
+    bool IsActiveDocument { get; }
```

Comment:
what will this be used for?

Reply:
do we need this if the searchresult already exposes NavigableItem?  could we not just use that to see if the result's doc-id is the same as the original id?

Reply:
I'm not sure how I'd figure that out from within RoslynNavigateToSearchCallback

Reply:
i would *think* the active document would be available/captured into the callback.  and its doc id could be compared against the doc id in NavigableItem.

Reply:
I didn't see a way to do this, so resolving. But please reopen if I'm missing something.